__tests__/run_seed.test.js

```diff
@@ -82,6 +82,92 @@ describe("App testing", () => {
         });
     });
   });
+  describe("GET: queries", () => {
+    test("Returns a 200 status and accepts a sort-by query, which sorts articles by any valid column, sorting by created_at by default", () => {
+      return request(app)
+        .get(`/api/articles?sort_by=title`)
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.articles[0].author).toBe("rogersop");
+          expect(body.articles[body.articles.length - 1].author).toBe(
+            "icellusedkars"
+          );
+        });
+    });
+    test("responds with a 404 error when passed a sort_by column query where the passed column does not exist on the database", () => {
+      return request(app)
+        .get(`/api/articles/?sort_by=rainbows`)
+        .expect(404)
+        .then(({ body: { msg } }) => {
+          expect(msg).toBe("Not found!");
+        });
+    });
+    test("Returns a 200 status and accepts a sort-by query sorting by created_at by default", () => {
+      return request(app)
+        .get(`/api/articles`)
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.articles[0].created_at).toBe("2020-11-03T09:12:00.000Z");
+          expect(body.articles[body.articles.length - 1].created_at).toBe(
+            "2020-06-06T09:10:00.000Z"
+          );
+        });
+    });
+    test("200 : accepts an order query, with default set to descending order", () => {
+      return request(app)
+        .get(`/api/articles?sort_by=title&order=ASC`)
+        .expect(200)
+        .then(({ body: { articles } }) => {
+          expect(articles[0].author).toBe("icellusedkars");
+          expect(articles[articles.length - 1].author).toBe("rogersop");
+        })
+        .then(() => {
+          return request(app)
+            .get(`/api/articles?sort_by=title`)
+            .expect(200)
+            .then(({ body }) => {
+              expect(body.articles[0].author).toBe("rogersop");
+              expect(body.articles[body.articles.length - 1].author).toBe(
+                "icellusedkars"
+              );
+            });
+        });
+    });
+    test("Responds with a 200 status and accepts a topic query", () => {
```

Comment:
What about a topic that exists but has no associated articles? Take a look in the topics data file for clues!